Automattic/sensei
# Update free Lesson patterns.

Fixes partially #5210

### Changes proposed in this Pull Request
* Update existing lesson patterns.
* Add: default lesson.
* Add: default lesson with Quiz.
* Add: Zoom/Meeting

### Testing instructions
- Create a new Lesson.
- Check that all the patterns appear and look as they should.

### Screenshot / Video
Screenshots using Divi theme.

#### List of Lesson Patterns (only free ones)
![image](https://user-images.githubusercontent.com/799065/172837428-c2d40406-4bad-4e8d-8efa-4ad4cfb02a1c.png)

#### Default (*new)
![image](https://user-images.githubusercontent.com/799065/172837560-b911b7fd-d5ff-4380-bd65-fcd16ded7d26.png)

#### Default with Quiz (*new)
![image](https://user-images.githubusercontent.com/799065/172837724-605372ff-b12a-4774-9c20-04aec0ff158d.png)

#### Video Lesson
![image](https://user-images.githubusercontent.com/799065/172837867-33c6abcb-f7ae-4c2a-a394-abd9f7cb2bf5.png)

#### Discussion Question Lesson
![image](https://user-images.githubusercontent.com/799065/172837973-473d9eeb-f037-4187-b0a7-ca5c58312797.png)

#### Files to Download
![image](https://user-images.githubusercontent.com/799065/172838207-23fd697a-6452-4a75-a691-bcd1059f9baa.png)

#### Zoom/Meeting (*new)
![image](https://user-images.githubusercontent.com/799065/172838611-4d910828-d76e-4a7c-bfe7-b788180438cd.png)

### Known bugs
- [x] Lesson actions should be at the bottom also for the Default with Quiz pattern. FIXED! ✅ 
- [ ] The default **teamplate** for the lessons is different from the Default pattern. @renatho is working on a solution to reuse the pattern in the template for the Courses. We will reuse the same for the Lessons.
